src/controllers/dynakube/version/version.go

```diff
@@ -8,7 +8,7 @@ import (
 
 	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
 	"github.com/Dynatrace/dynatrace-operator/src/dockerconfig"
-	"github.com/Dynatrace/dynatrace-operator/src/kubeobjects"
+	timeutil "github.com/Dynatrace/dynatrace-operator/src/time"
```

Comment:
If we have other time utils (there, I said it :-)) in the future, we need to make another package and have conversation again the same  again. It could be worse, e.g. `timehelper`

Reply:
I don't see any reason to have other timeutils that couldn't be part of the `timeprovider` as a method